### PR TITLE
Incorporate examples into Sphinx.

### PR DESCRIPTION
**Pull request by mossblaser**
How does this look to you, @mundya?

### CODE REVIEW

File: 01_hello_world/hello.c [hunk 1]
```diff
@@ -2,15 +2,9 @@
...
 
-void c_main(void) {
-    // This nearly-standard printf function prints its message to one of many
-    // locations. In this case we're printing to the "IO buffer" which is an
-    // area of on-chip memory automatically allocated by the system software.
-    // We will later read the contents of the IO buffer back from the host in
-    // order to display it to the user.
-    io_printf(IO_BUF, "Hello, world!\n");
+void c_main(void)
+{
```
Comment (mundya) [inline 01_hello_world/hello.c:8]: :smile:

**Comment by mundya** [comment]
LGTM - I guess we could move this into the Rig repo if we so wished?

**Comment by mundya** [comment]
Or at least point ReadTheDocs at it?

File: 01_hello_world/README.rst [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+.. note::
+    
+    For those who don't have the ARM cross-compiler installed a precompiled binary
+    is included with this example.
+
+Host-side application
+---------------------
+
+Now that we have our compiled binary we must boot our SpiNNaker machine, load
+our application onto a core and then read back the binary. We could do this
```
Comment (mundya) [inline 01_hello_world/README.rst]: read back the IOBUF?

Comment (mossblaser) [inline 01_hello_world/README.rst]: Yes, of course!

**Comment by mossblaser** [comment]
> I guess we could move this into the Rig repo if we so wished?

I think it is better as its own repo for ease of checking out

> Or at least point ReadTheDocs at it?

Absolutely!

**Pull request merged**